src/targets/gpu/device/include/migraphx/gpu/device/reduce.hpp

```diff
@@ -25,6 +25,12 @@
 #ifndef MIGRAPHX_GUARD_RTGLIB_DEVICE_REDUCE_HPP
 #define MIGRAPHX_GUARD_RTGLIB_DEVICE_REDUCE_HPP
 
+#if defined(__GFX10__) || defined(__GFX11__) || defined(__GFX12__) 
+    #define __MIGRAPHX_WAVEFRONT_SIZE 32 
```

Comment:
Dont prefix the macro with a double underscore `__`. It should just be `MIGRAPHX_WAVEFRONT_SIZE`.